rasa/core/policies/ted_policy.py

```diff
@@ -773,7 +866,8 @@ def persist(self, path: Union[Text, Path]) -> None:
             return
 
         model_path = Path(path)
-        tf_model_file = model_path / f"{SAVE_MODEL_FILE_NAME}.tf_model"
+        metadata_filename = self._metadata_filename()
```

Comment:
Why do we call this "metadata"? To me that sounds like "date of training" or something, not its weights or unlikely-intent-thresholds.

Reply:
+1

Reply:
Is `model_file_name` okay?

Reply:
Fine by me